### ISSUE TITLE
PDFUA-11: Link annotation not inside link structure element

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/PDFA-TEST#11]
Jira issue originally created by user @:

A link [annotation](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#annotation) is used, but not inside link structure element

**Comment by zakkinsey** [comment]
Comment created by @:

{color:#000000}Evaluation with Acrobat and JAWS {color}

{color:#000000} {color}

{color:#000000}Adobe Acrobat DC (19.010.10059){color}

{color:#000000}JAWS Version 2018.1811.2 ILM{color}

{color:#000000}Windows 10{color}

{color:#000000} {color}

{color:#000000}When needed to exemplify what JAWS is speaking, I’m using the same notation used in our guides for tagging psudo code. For instance, <H1> \{foobar} means that JAWS speaks the text foobar as a heading level 1.{color}

{color:#000000} {color}

Broken indeed, JAWS does not announce it as a link.

**Comment by zakkinsey** [comment]
Comment created by @pkopicar:

Duplicates PDFUA-146 ?
